frontend/public/components/utils/resource-log.tsx

```diff
@@ -234,150 +274,311 @@ export const LogControls: React.FC<LogControlsProps> = ({
       </Tooltip>
     );
   };
+
+  const renderPodLogLinks = () =>
+    _.map(_.sortBy(podLogLinks, 'metadata.name'), (link) => {
+      const namespace = resource.metadata.namespace;
+      const namespaceFilter = link.spec.namespaceFilter;
+      if (namespaceFilter) {
+        try {
+          const namespaceRegExp = new RegExp(namespaceFilter, 'g');
+          if (namespace.search(namespaceRegExp)) {
+            return null;
+          }
+        } catch (e) {
+          // eslint-disable-next-line no-console
+          console.warn('invalid log link regex', namespaceFilter, e);
+          return null;
+        }
+      }
+      const url = replaceVariables(link.spec.hrefTemplate, {
+        resourceName: resource.metadata.name,
+        resourceUID: resource.metadata.uid,
+        containerName,
+        resourceNamespace: namespace,
+        resourceNamespaceUID: namespaceUID,
+        podLabels: JSON.stringify(resource.metadata.labels),
+      });
+      return (
+        <>
+          {isMobile ? (
+            <DropdownItem key={link.metadata.uid}>
+              <ExternalLink href={url} text={link.spec.text} dataTestID={link.metadata.name} />
+              <Divider
+                orientation={{
+                  default: 'vertical',
+                }}
+              />
+            </DropdownItem>
+          ) : (
+            <React.Fragment key={link.metadata.uid}>
+              <ExternalLink href={url} text={link.spec.text} dataTestID={link.metadata.name} />
+              <Divider
+                orientation={{
+                  default: 'vertical',
+                }}
+              />
+            </React.Fragment>
+          )}
+        </>
+      );
+    });
+
   const label = t('public~Debug container');
+  const renderLogViewerSearch = () => (
+    <>
+      <div className="co-toolbar__item">
+        <LogViewerSearch
+          onFocus={() => {
+            if (status === STREAM_ACTIVE) {
+              toggleStreaming();
+            }
+          }}
+          placeholder="Search"
+          minSearchChars={0}
+        />
+      </div>
+      {showDebugAction(resource, containerName) && !isWindowsPod(resource) && (
+        <Link
+          to={`${resourcePath(
+            'Pod',
+            resource.metadata.name,
+            resource.metadata.namespace,
+          )}/containers/${containerName}/debug`}
+          data-test="debug-container-link"
+        >
+          {label}
+        </Link>
+      )}
+      {showDebugAction(resource, containerName) && isWindowsPod(resource) && (
+        <Tooltip
+          content={t('public~Debug in terminal is not currently available for windows containers.')}
+        >
+          <span className="text-muted">{label}</span>
+        </Tooltip>
+      )}
+    </>
+  );
+
+  const renderShowFullLog = () => (
+    <Tooltip
```

Comment:
```suggestion
    <div>
        <Tooltip
      content={t('public~Select to view the entire log. Default view is the last 1,000 lines.')}
    >
        <Checkbox
          label={t('public~Show full log')}
          id="showFullLog"
          data-test="show-full-log"
          isChecked={isShowFullLog}
          data-checked-state={isShowFullLog}
          onChange={(_event, checked: boolean) => {
            toggleShowFullLog(checked);
          }}
        />
      </Tooltip>
   </div>
```
Fixes an issue where the spacing after "log" collapses (see screenshot below).
<img width="182" alt="Screenshot 2023-11-29 at 9 30 30 AM" src="https://github.com/openshift/console/assets/895728/70458a63-5d28-4d17-aa86-4a0f059b5f47">